arex-instrumentation/redis/arex-redission-v315/src/main/java/io/arex/inst/redisson/v315/wrapper/CommandSyncServiceAdviceWrapper.java

```diff
@@ -0,0 +1,149 @@
+        Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null, Arrays.toString(params),
+            () -> super.readAsync(client, name, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readAsync(RedisClient client, byte[] key, Codec codec, RedisCommand<T> command,
+        Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), Arrays.toString(key),
+            Arrays.toString(params),
+            () -> super.readAsync(client, key, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readAsync(RedisClient client, Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null,
+            Arrays.toString(params),
+            () -> super.readAsync(client, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readAsync(String key, Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), key,
+            Arrays.toString(params),
+            () -> super.readAsync(key, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readAsync(byte[] key, Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), Arrays.toString(key),
+            Arrays.toString(params),
+            () -> super.readAsync(key, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readAsync(MasterSlaveEntry entry, Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null,
+            Arrays.toString(params), () -> super.readAsync(entry, codec, command, params));    }
+
+    @Override
+    public <T, R> RFuture<R> readRandomAsync(Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null, Arrays.toString(params),
+            () -> super.readRandomAsync(codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> readRandomAsync(MasterSlaveEntry entry, Codec codec, RedisCommand<T> command,
+        Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null, Arrays.toString(params),
+            () -> super.readRandomAsync(entry, codec, command, params));
+    }
+
+    @Override
+    public <T, R> RFuture<R> writeAsync(MasterSlaveEntry entry, Codec codec, RedisCommand<T> command, Object... params) {
+        return RedissonWrapperCommon.delegateCall(redisUri, command.getName(), null, Arrays.toString(params),
+            () -> super.writeAsync(entry, codec, command, params));    }
```

Comment:
Previously, redis set operations did not record the value, but now the writeAsync method will record the value, please confirm this!

```java
io.arex.inst.redisson.v3.wrapper.RedissonBucketWrapper#setAsync(V)

    @Override
    public RFuture<Void> setAsync(V value) {
        return RedissonWrapperCommon.delegateCall(redisUri, RedisCommands.SET.getName(), this.name,
            () -> super.setAsync(value));
    }

org.redisson.RedissonBucket#setAsync(V)

    @Override
    public RFuture<Void> setAsync(V value) {
        if (value == null) {
            return commandExecutor.writeAsync(getRawName(), RedisCommands.DEL_VOID, getRawName());
        }

        return commandExecutor.writeAsync(getRawName(), codec, RedisCommands.SET, getRawName(), encode(value));
    }
```